src/pages/CatEdit.js

```diff
@@ -1,11 +1,82 @@
 import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import { Form, FormGroup, Input, Label, Button } from "reactstrap";
+import "./CatEdit.css";
+import "../App.css";
 
 export default class CatEdit extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      updateCat: {
+        name: "",
+        age: "",
+        enjoys: "",
+        img: "",
+      },
```

Comment:
Did you want to pass in props to set the current state and display that info to the user before they edit the cat?